src/core/tranche/slicer.sol

```diff
@@ -35,15 +35,22 @@ contract Slicer is DSNote {
     modifier auth { require(wards[msg.sender] == 1); _; }
     
     // --- Data ---
-    TrancheManagerLike public trancheManager;
+    AssessorLike public assessor;
     ReserveLike public reserve;
+    address operator;
 
-    constructor(address trancheManager_, address reserve_) public {
+    constructor(address assessor_, address reserve_) public {
         reserve = ReserveLike(reserve_);
-        trancheManager = TrancheManagerLike(trancheManager_);
+        assessor = AssessorLike(assessor_);
         wards[msg.sender] = 1;
     }
 
+    function file(bytes32 what, address address_) public note auth {
+        if (what == "operator") {
```

Comment:
i think contract dependencies should use the `depend` method, file is just for state variables

Reply:
this is a state variable, passed to the assessor as the trancheId